src/lammpsio/data.py

```diff
@@ -129,6 +137,18 @@ def create(cls, filename, snapshot, atom_style=None):
             if snapshot.box.tilt is not None:
                 f.write("{} {} {} xy xz yz\n".format(*snapshot.box.tilt))
 
+            if snapshot.bonds is not None:
```

Comment:
Since we have these convenience methods, we should use them here:
```suggestion
            if snapshot.has_bonds():
```